DESCRIPTION

```diff
@@ -6,7 +6,7 @@ Authors@R: c(person("Rich", "FitzJohn", role = c("aut", "cre"),
              person("Imperial College of Science, Technology and Medicine",
                     role = "cph"))
 Title: Vault Client for Secrets and Sensitive Data
-Version: 1.1.4
+Version: 1.1.5
```

Comment:
No - I'll release a version with this and some other fixes once the CRAN summer closure is over (so in ~3 weeks). Probably I'll release it as 1.2.0

In our internal processes we bump on each merge to master/main because internally we are continually releasing - this way we can track back from the version that a user installs (from our r-universe, from our old drat repo, or from github) back to the source

Reply:
sounds great! Thanks a bunch for maintaining this package and the new LDAP support! ✨

Reply:
hey @richfitz ! Any updates here on a release timeline? No immediate pressure, but would love to start using 😄

Reply:
Oh I am sorry, this has dropped off my radar.  I can do it now - it looks like it will pass CRAN, but I am going to have no ability to get an update done in the next couple of weeks which means if there is a problem it would get bounced from CRAN. So my plan is to submit this second week of November

Reply:
No problem - sounds great!

Reply:
This is submitted to CRAN and will update there in the next day or wo

Reply:
On CRAN now as 1.2.0: https://cran.r-project.org/web/packages/vaultr/index.html